src/app/theme/theme.ts

```diff
@@ -53,10 +55,54 @@ export const theme: DefaultTheme = {
 
 // Global styling for default elements
 export const GlobalStyle = createGlobalStyle`
+  body { 
+    margin: 0;
+    overflow: hidden;
+  }
+
   :root {
     color: ${(p) => p.theme.colors.BODY};
     font-family: ${(props) => props.theme.fonts.default};
     font-size: ${(props) => props.theme.fontSizes[2]};
     line-height: ${(props) => props.theme.lineHeights[1]};
   }
 `;
+
+// FROM CORE, DELETE WHEN COMPONENT SHARING
+const FLEX_RECIPES = {
+  align: {
+    alignItems: 'center'
+  },
+  center: {
+    justifyContent: 'center',
+    alignItems: 'center'
+  }
+};
+
+const FLEX_VARIANTS = {
+  rowAlign: {
+    display: 'flex',
+    flexDirection: 'row',
+    ...FLEX_RECIPES.align
+  },
+  rowCenter: {
+    display: 'flex',
+    flexDirection: 'row',
+    ...FLEX_RECIPES.center
+  },
+  columnAlign: {
+    display: 'flex',
+    flexDirection: 'column',
+    ...FLEX_RECIPES.align
+  },
+  columnCenter: {
+    display: 'flex',
+    flexDirection: 'column',
+    ...FLEX_RECIPES.center
+  }
+};
+
+export type FlexVariants = keyof typeof FLEX_VARIANTS;
+export const flexVariants = variant({
+  variants: FLEX_VARIANTS
+});
```

Comment:
Migrate those variant to the theme, with rebass it's easier to ship them with the theme (https://rebassjs.org/variants)